src/instrumentations/xhr.ts

```diff
@@ -47,11 +47,14 @@ import type { PropagateTraceHeaderCorsUrls } from '@opentelemetry/sdk-trace-web/
 
 const parseUrl = (url: string) => new URL(url);
 
+const MAX_BODY_LENGTH = 5 * 1024; // 5KB
```

Comment:
5k seems a bit too strict. maybe something like 2MB to start with is okay

Reply:
updated